examples/04_local_pickup/01_get_pickup_availability_for_variants/query.graphql

```diff
@@ -1,25 +1,21 @@
-query GetPickUpAvailability {
-  product(first: 1) {
-    edges {
-      node {
-        variants(first: 1) {
-          edges {
-            node {
-              storeAvailability(first: 1) {
-                edges {
-                  node {
-                    available
-                    pickUpTime
-                    location {
-                      name
-                    }
-                  }
-                }
-              }
+# Before sending this request, please make sure your app has unauthenticated_read_product_pickup_locations scope, and store pick up has been enabled.
+
+# You can also use "id" to replace "handle" as argument to query product. 
+
+query GetStoreAvailability {
+  product(handle: "Orange") {
+    variantBySelectedOptions(selectedOptions: {name: "Weight", value: "2 lb"}) {
+      storeAvailability(first: 1) {
+        edges {
+          node {
+            available
+            pickUpTime
+            location {
+              name
             }
           }
         }
       }
     }
   }
-}
+}
```

Comment:
Hey @emmajxli - I think the inputs could be dynamic here to keep with the principles of the SFAPI learning kit -  Something like:

```
query GetStoreAvailability($handle: String! $name: String! $value:String!) {
  product(handle: $handle) {
    variantBySelectedOptions(selectedOptions: {name: $name, value: $value}) {
      storeAvailability(first: 1) {
        edges {
          node {
            available
            pickUpTime
            location {
              name
            }
          }
        }
      }
    }
  }
}
```
(with variable file added) 

Also, nitpick, on my PR's, I'd included a link to the docs when there's some requirements they have to fulfill before running the SFAPI query :)

Reply:
variables.json file added for getStoreAvailability query